### DISCUSSION TITLE
Jóhann fer til Danmerkur - 2024/7/4

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#290] [vefsida-ummæli]
# Jóhann fer til Danmerkur - 2024/7/4

Við misstum mikilvægan starfsmann

https://www.biskupsreglan.com/frettir/Danm%C3%B6rk/H%C3%A6ttufer%C3%B0in-til-Danmerkur

**Comment by TJ478** [comment]
Bakasaga...
